app/views/assessments/activity/activity-2.html.erb

```diff
@@ -7,25 +7,12 @@
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
             <div class="govuk-caption-l">Activity checks</div >
             <h1 class="govuk-fieldset__heading">
-              Have you found interactions that you’d expect that person to have?
+              Over what period of time did you find the interactions?
```

Comment:
This question still doesn't really make sense to me, let's run it by Tom once again before we land this.